design-reference/architecture/aad/easyauth.md

```diff
@@ -0,0 +1,35 @@
+# Purpose/Description
```

Comment:
Grouping comments in this section into a single comment:
 - Can you provide a link to documentation on *easy auth* so that this document is more stand-alone?
 - [ Nit about structure of designs ] This section starts off with an implementation plan (*we do this by extending foo and bar modules*) but its difficult as somebody who is not familiar with this technology to understand what is being accomplished and why. I think a more compelling structure may start with the problem statement (you do this), move into who you are targeting as a customer (in broad strokes -- not specifically who they are) for this design is and why they want to do this on azure. Then discussing about what built-in best practices exist in azure (i.e., AD) and then finally how the existing modules can be adapted to fulfill the use cases you are solving for. This pre-empts a lot of questions that you will get in a typical design review session. In short, this assumes that easy-auth is the right option without providing any compelling evidence of why.